src/main/java/pojos/Descriptions.java

```diff
@@ -0,0 +1,387 @@
+package pojos;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Descriptions {
```

Comment:
it seems the model classes in "record" package are same as with the ones in pojo package. (for example you have 2 Descriptions classes with same fields. is there a reason for that?)

Reply:
remove record.